src/apps/content-editor/src/app/views/ItemList/SetRow/PublishStatusCell/PublishStatusCell.js

```diff
@@ -2,35 +2,26 @@ import React from "react";
 import cx from "classnames";
 
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faClock, faEye } from "@fortawesome/free-solid-svg-icons";
+import { faClock, faEye, faCircle } from "@fortawesome/free-solid-svg-icons";
 import { Url } from "@zesty-io/core/Url";
 
 import styles from "./PublishStatusCell.less";
 export const PublishStatusCell = React.memo(function PublishStatusCell(props) {
   if (props.type === "dataset") {
     return (
-      <span className={cx(styles.PublishStatusCell)}>
+      <Url className={cx(styles.PublishStatusCell)} href={`${props.url}`}>
```

Comment:
This will need to be the AppLink component if we want this to be linkable in app and not cause a browser page load